src/flux/step-panel/policies.coffee

```diff
@@ -0,0 +1,85 @@
+freeResponse =
+  name: 'free-response'
+  optional: 'checkQuestionFormat'
+  passCheck: ['free_response']
+  actions: [
+    'fillFreeResponse'
+    'saveFreeResponse'
+  ]
+
+multipleChoice =
+  name: 'multiple-choice'
+  optional: 'checkQuestionFormat'
+  passCheck: ['answer_id','is_completed']
+  actions: [
+    'pickMultipleChoice'
+    'saveMultipleChoice'
+  ]
+
+review =
+  name: 'review'
+  actions: [
+    'clickContinue'
+  ]
+
+view =
+  name: 'view'
+  actions: [
+    'clickContinue'
+  ]
+
+stepNoFeedback = [
+  freeResponse
+  multipleChoice
+]
+
+stepFeedbackImmediate = [
+  freeResponse
+  multipleChoice
+  review
+]
+
+stepViewOnly = [
+  view
+]
+
+# Policy Shape
```

Comment:
I think it could just be a vanilla `.coffee` file since it does not talk to the server; but that can be a cleanup in the future